File: platform/sdk/nodejs/examples/cloud/cloud-019_quota_50_per_min.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const result = client.quota50PerMin(30);
if (!result.withinLimit || result.remaining !== 20) process.exit(1);
console.log("XJ-OK: cloud-019_quota_50_per_min");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-019_quota_50_per_min.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
result = client.quota50PerMin(30)
if not result["withinLimit"] or result["remaining"] != 20:
    sys.exit(1)
print("XJ-OK: cloud-019_quota_50_per_min")
sys.exit(0)
